Improved: List and Grid (OFBIZ-11345)

PierreSmits opened the pull request:
According to the definition in widget-form.xsd the use of a combination of a form with type="list" is deprecated in favour of a grid.
Refactor various list forms into grids.
Refactor various list form references in screens.

Modified:
InvoiceScreens.xml: from form ref to grid ref , additional cleanup
InvoiceForms.xml: from form definition with list ref to grid definition with list ref, additional clean-up

PierreSmits committed:
Improved: List and Grid (OFBIZ-11345)

According to the definition in widget-form.xsd the use of a combination of a form with type="list" is deprecated in favour of a grid.
Refactor various list forms into grids.
Refactor various list form references in screens.

Modified:
InvoiceScreens.xml: from form ref to grid ref , additional cleanup
InvoiceForms.xml: from form definition with list ref to grid definition with list ref, additional clean-up

JacquesLeRoux reviewed (changes requested):
Pierre, you forgot ListInvoicePaymentInfo in InvoiceScreens.xml

PierreSmits commented:
Hi Jacques,

Thanks for the feedback. I'll look into it.

PierreSmits committed:
Improved: List and Grid (OFBIZ-11345)

Modified:
InvoiceScreens.xml: missed ref to ListInvoicePaymentInfo in InvoiceScreens.xml
